checador/app/models/company.rb

```diff
@@ -0,0 +1,2 @@
+class Company < ApplicationRecord
```

Comment:
What do you think of adding some validations of the model here? I want to think that you don't need a Company without a `name`, and also add some specs for this model